server.js

```diff
@@ -25,6 +27,13 @@ console.info(`info: IS DEV: ${process.env.IS_DEV}`);
 app.use('/favicon.png',   express.static(__dirname + '/app/images/favicon.png', { maxAge: 24*60*60*1000 }));
 app.use('/app',           express.static(__dirname + '/app'     , { setHeaders: setCustomCacheControl }));
 app.use('/app/libs',      express.static(__dirname + '/node_modules/@bower_components', { setHeaders: setCustomCacheControl }));
+
+app.get('/app/authorize.html', (req, res) => {
+  var trustedDomains = domains.map(domain=>`/(^|.+\\.)${_.escapeRegExp(domain)}$/i`);
+  setCustomCacheControl(res);
```

Comment:
there is not cache invalidation on authorize.html so its better off without any cachecontrol

Reply:
I reuse the same cacheControl setting that was on `authorize.html` before (line 28 of same file

https://github.com/scbd/accounts.cbd.int/pull/53/files/f9767df6140a142968af6628d4b9286ec3d5d19a#diff-a4c65ede64197e1a112899a68bf994485b889c4b143198bac4af53425b38406fR28